lib/Config/ServicesChecker.php

```diff
@@ -116,4 +117,17 @@ function isPremiumKeyValid($display_error_notice = true) {
 
     return false;
   }
+
+  function isFromEmailAuthorized($display_error_notice = true) {
```

Comment:
Again here: why did you use the optional `$display_error_notice` argument if you always pass the default value? And also boolean arguments are a bad practice. If you are reading it somewhere you see only: `$checker->isFromEmailAuthorized(true)` and you don't know what `true` means. You need to go to the definition to see.  It is always better to have two separate functions than creating one with a boolean argument.

The use of `WPNotice` here is a perfect example. What does `'', false, false` mean? without going there, there is no way of knowing.

Reply:
Removed that argument altogether. The idea was to be consistent with other functions, but I agree that here it doesn't make sense.